supernode/services/artworkregister/node.go

```diff
@@ -33,7 +33,7 @@ func (nodes Nodes) ByID(id string) *Node {
 func (nodes *Nodes) Remove(id string) {
 	for i, node := range *nodes {
 		if node.ID == id {
-			*nodes = append((*nodes)[:i], (*nodes)[:i+1]...)
```

Comment:
@levkoburburas 
I found a bug on this line. actually the old code adding node instead remove node from nodes list. Let me know if i am wrong

Reply:
the code is refactored, need u help to update the unit test for the p2p module in the future

Reply:
ok sure